internal/services/aws_reservation_service.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
-	logger := ctxval.Logger(r.Context())
+	logger := *ctxval.Logger(r.Context())
```

Comment:
Just wondering, we didn't use the '*' until now, why did you add it? 
it seems like the compiler knows to reference it like the logger object and not as a pointer.

Reply:
That is indeed suspicions, I did this on purpose (see my comment below).

It would have been dangerous, but I know for fact that this function never returns `nil`: https://github.com/RHEnVision/provisioning-backend/blob/main/internal/ctxval/context_values.go#L34